libtransmission/rpcimpl.cc

```diff
@@ -317,7 +317,7 @@ void addFileStats(tr_torrent const* tor, tr_variant* list)
         tr_variant* d = tr_variantListAddDict(list, 3);
         tr_variantDictAddInt(d, TR_KEY_bytesCompleted, file.have);
         tr_variantDictAddInt(d, TR_KEY_priority, file.priority);
-        tr_variantDictAddBool(d, TR_KEY_wanted, file.wanted);
+        tr_variantDictAddInt(d, TR_KEY_wanted, file.wanted);
```

Comment:
This appears to be a bool in both doc and code in 3.00 and 4.0.x so we should probably leave this alone?

Reply:
Yeah, this one is fine to leave alone, from compat standpoint.